webapp/app.py

```diff
@@ -163,6 +163,13 @@ def examples_index():
     )
 
 
+@app.route("/docs/examples/patterns/notifications/toast.json")
```

Comment:
I don't know exactly either, but something like that could work?

```
@app.route("/<path:json_path>.json")
def json_examples(json_path):
```

And then you could use `json_path` to retrieve the file (I guess you would need to add `.json` to it for full file name.

Try to check some other examples where part of URL is passed to the function like in line 180:

```
@app.route("/docs/examples/standalone/<path:example_path>")
def standalone_example(example_path):
```

Reply:
But if you load them dynamically in the same endpoint, are all the json files going to be in the same folder? Wouldn't it make sense to separate them like they are now?

Reply:
The following works for me:

```python
@app.route("/<path:json_path>.json")
def json_examples(json_path):
    try:
        return flask.send_file(f"{app.template_folder}/{json_path}.json")
    except FileNotFoundError:
        return flask.abort(404)
```